line-sdk/src/main/java/com/linecorp/linesdk/openchat/ui/ProfileInfoFragment.kt

```diff
@@ -0,0 +1,91 @@
+package com.linecorp.linesdk.openchat.ui
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProviders
+
+import com.linecorp.linesdk.R
+import com.linecorp.linesdk.databinding.ProfileInfoFragmentBinding
+import com.linecorp.linesdk.openchat.TextUpdateWatcher
+import kotlinx.android.synthetic.main.profile_info_fragment.displayNameEditText
+import kotlinx.android.synthetic.main.activity_create_open_chat.toolbar
+
+class ProfileInfoFragment : Fragment() {
+
+    private lateinit var binding: ProfileInfoFragmentBinding
+
+    companion object {
+        fun newInstance() = ProfileInfoFragment()
+    }
```

Comment:
According to the coding convention, the companion object needs to be placed at the bottom of this class.